src/Currencies.jl

```diff
@@ -55,12 +55,19 @@ Returns the ISO 4217 name associated with this value
 """
 function name end
 
-symbol(::Currency{S}) where {S} = S
 currency(S::Symbol) = _currency_data[S][1]
 unit(S::Symbol) = _currency_data[S][2]
 code(S::Symbol) = _currency_data[S][3]
 name(S::Symbol) = _currency_data[S][4]
 
+symbol(::Type{Currency{S}}) where {S} = S
+currency(::Type{Currency{S}}) where {S} = currency(S)
```

Comment:
This should be `currencty(::Type{C}) where {C<:Currency} = C`, I think.

Reply:
If we do this, we get a currency `Type`. The way I have it there, we get a currency instance. Since it is a singleton, there is not a big difference and it would be a little more consistent with `instrument` to return the type. I'm ok either way, but do lean a bit more toward return an instance from `currency` and a type from `instrument`.

Reply:
Come to think of it, I believe we previously decided to let `currency` return an instance and I think we can stick with that (see [here](https://github.com/JuliaFinance/Currencies.jl/blob/ef/src/Currencies.jl#L58)).